### PR TITLE
Add release notes for listdescriptors RPC

### PR DESCRIPTION
**Pull request by S3RK**
Original PR is #20226

### CODE REVIEW

**Comment by S3RK** [comment]
cc @jonatack

**Comment by laanwj** [comment]
ACK 1ac5bc03c6d385cdc62e1d3560dfbcf3fed4d3f5

File: doc/release-notes.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ Tools and Utilities
 Wallet
 ------
 
+- New `listdescriptors` RPC is available to inspect contents of a descriptor-enabled wallet.
+  The RPC returns public versions of all imported descriptors, including their timestamp and flags.
+  For ranged descriptor it also returns range boundaries and the next index to generate addresses from.
```
Comment (jonatack) [inline doc/release-notes.md:98]:
```suggestion
  For ranged descriptor it also returns range boundaries and the next index to generate addresses from. (#20226)
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -93,6 +93,10 @@ Tools and Utilities
 Wallet
 ------
 
+- New `listdescriptors` RPC is available to inspect contents of a descriptor-enabled wallet.
```
Comment (jonatack) [inline doc/release-notes.md:96]:
```suggestion
- A new `listdescriptors` RPC is available to inspect the contents of descriptor-enabled wallets.
```

File: doc/release-notes.md (hunk 1 above)
Comment (jonatack) [inline doc/release-notes.md:98]:
```suggestion
  For ranged descriptors, it also returns the range boundaries and the next index to generate addresses from.
```

**Review by jonatack — COMMENTED** [review]
A few minor suggestions below. Do you plan to address https://github.com/bitcoin/bitcoin/pull/20226#discussion_r565608783?

**Comment by S3RK** [comment]
> I tried so hard and got so far
> But in the end there were still a few minor suggestions...  

Thanks jonatack, hope one day I'll get my articles right 😄 

> Do you plan to address [#20226 (comment)](https://github.com/bitcoin/bitcoin/pull/20226#discussion_r565608783)?

Done — #21063

**Comment by jonatack** [comment]
ACK 51f3752 

Thanks for updating. No worries :)

**Review by jonasschnelli — APPROVED** [review]
ACK 51f3752fbeee09d025db33e154bb2efff9e20837

**Pull request merged**